neo/rawio/plexonrawio.py

```diff
@@ -363,8 +389,8 @@ def _segment_t_start(self, block_index, seg_index):
     def _segment_t_stop(self, block_index, seg_index):
         t_stop = float(self._last_timestamps) / self._global_ssampling_rate
         if hasattr(self, "_signal_length"):
-            for stream_id in self._signal_length:
-                t_stop_sig = self._signal_length[stream_id] / self._sig_sampling_rate[stream_id]
+            for stream_index in self._signal_length:
```

Comment:
My only comment is that you are iterating through the keys, but it is not clear as a developer that this is a dict and not a list. I think iterating through the `keys()` would make it clear the object is a dict. But this is just an  optional comment. Switching it to `stream_index` fits with the current data, so I think that is a great semantic fix.

Reply:
This was already there, I just change the wonrgly named variable stream_id to stream_index.

But I agree with you that it would be better to be explicit and do self._signal_length.keys().

Reply:
No I know you were just fixing it! But when it's on the diff it gets spotted :)

Reply:
Changed.